migrations/20191002020117-create-user.js

```diff
@@ -9,10 +9,15 @@ module.exports = {
         type: Sequelize.INTEGER
       },
       email: {
-        type: Sequelize.STRING
+        type: Sequelize.STRING,
+        allowNull: false,
+        isEmail: true,
+        isUnique: true
```

Comment:
I love what you've done here -- I'm guessing that these are what would be used for validations in Rails? Really cool piece of code that I'll try to implement later in my project if I have the opportunity to do!